spring-cloud-modules/spring-cloud-openfeign/src/main/java/com/baeldung/cloud/openfeign/defaulterrorhandling/client/ProductClient.java

```diff
@@ -0,0 +1,17 @@
+package com.baeldung.cloud.openfeign.defaulterrorhandling.client;
+
+import com.baeldung.cloud.openfeign.defaulterrorhandling.config.FeignConfig;
+import com.baeldung.cloud.openfeign.defaulterrorhandling.model.Product;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+@FeignClient(name = "product-client", url = "http://localhost:8081/product/", configuration = FeignConfig.class)
+public interface ProductClient {
+
+    @RequestMapping(value = "{id}", method = RequestMethod.GET, produces = "application/json")
```

Comment:
`produces`? or `consumes`?

Reply:
Removed that for now. Works without that as well.